### ISSUE TITLE
Unable To Disable Invitations For Group Assignments

### ISSUE THREAD

**Issue by ahanmal** [education/classroom#2555]
I am unable to disable invitations for group assignments. Toggling the checkbox results in the loading indicator loading indefinitely. Looking at the XHR call, it appeared that the `toggle_invitations` endpoint was returning a 404. Looking closely at the request URL and the page URL, it appears that the page was hitting `.../assignments/ps7/toggle_invitations` instead of `.../group-assignments/ps7/toggle_invitations`. Sure enough, running a cURL on the fixed url resulted in a successful toggling of the invitations. 

TL;DR: The invitation endpoint for group assignments needs to be fixed so that it is `group-assignments` instead of `assignments`. 

Thanks!

**Comment by d12** [comment]
Thanks for the report, and nice debugging!! We've been a little tight on
engineering lately and haven't gotten to dig into this yet, but this looks
simple to fix. I'll update here when we've pushed something up to
production.

**Comment by ahanmal** [comment]
Thanks!

**Comment by d12** [comment]
@ahanmal All fixed up! Thanks for the report :)
